penguin/local_auth.py

```
@@ -0,0 +1,100 @@
+    *,
+    host: str | None = None,
+    port: int | None = None,
+) -> Path:
+    """Persist a loopback Penguin startup token for local launcher reuse."""
+    path = local_auth_token_path(base_url, host=host, port=port)
+    path.parent.mkdir(parents=True, exist_ok=True)
+    path.write_text(token.strip(), encoding="utf-8")
+    try:
+        path.chmod(0o600)
+    except OSError:
+        pass
```

Ruff (0.15.10)

[warning] Use `contextlib.suppress(OSError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(OSError): ...`

(SIM105)